lib/src/web/web_sqlite_open_factory.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:async';
+
+import 'package:sqlite3/wasm.dart';
+import 'package:sqlite3_web/sqlite3_web.dart';
+import 'package:sqlite_async/sqlite_async.dart';
+import 'package:sqlite_async/src/web/web_mutex.dart';
+
+import 'database.dart';
+
+Map<String, FutureOr<WebSqlite>> webSQLiteImplementations = {};
+
+/// Web implementation of [AbstractDefaultSqliteOpenFactory]
+class DefaultSqliteOpenFactory
+    extends AbstractDefaultSqliteOpenFactory<CommonDatabase> {
+  final Future<WebSqlite> _initialized;
+
+  DefaultSqliteOpenFactory(
+      {required super.path,
+      super.sqliteOptions = const SqliteOptions.defaults()})
+      : _initialized = Future.sync(() {
+          final cacheKey = sqliteOptions.webSqliteOptions.wasmUri +
+              sqliteOptions.webSqliteOptions.workerUri;
+
+          if (webSQLiteImplementations.containsKey(cacheKey)) {
+            return webSQLiteImplementations[cacheKey]!;
+          }
+
+          webSQLiteImplementations[cacheKey] = WebSqlite.open(
+            wasmModule: Uri.parse(sqliteOptions.webSqliteOptions.wasmUri),
+            worker: Uri.parse(sqliteOptions.webSqliteOptions.workerUri),
+          );
+          return webSQLiteImplementations[cacheKey]!;
+        });
+
+  @override
+
+  /// This is currently not supported on web
+  CommonDatabase openDB(SqliteOpenOptions options) {
+    throw UnimplementedError(
+        'Direct access to CommonDatabase is not available on web.');
+  }
+
+  @override
+
+  /// Currently this only uses the Drift WASM implementation.
+  /// The Drift SQLite package provides built in async Web worker functionality
+  /// and automatic persistence storage selection.
+  /// Due to being asynchronous, the under laying CommonDatabase is not accessible
```

Comment:
This also doesn't seem accurate anymore.

Reply:
Thanks! I've searched for and updated all mentions of Drift now.